Makefile.all

```diff
@@ -3,57 +3,67 @@
 # This file defines all packages available for building and defines their
 # dependencies, so that make can build them in parallel effectively.
 #
+# Each package that can be built by conan contains a `conanfile.py`
+# and a Makefile that includes `Makefile.package`.
+# By default all plugin packages are built.
+#
+# Use WITHOUT_PKGS variable on the command line to opt-out of specific
+# packages:
+#
+#     make WITHOUT_PKGS="plugins/demo_printer" package
+#
+# This extends the UNSELECT_PKGS variable. Set this variable directly
+# to override default omitted packages. Use make help to see which
+# these are. For example, if you want to export all vendor packages:
+#
+#	   make UNSELECT_VENDOR= export-vendor
+#
+# Use WITH_PKGS variable on the command line to include specific by-default
+# unselected packages:
+#
+#     make PLUGIN_PKGS= WITH_PKGS="plugins/minimator" package
+#
+# This extends the SELECT_PKGS variable. Set this variable directly
+# to explicitely specify exactly which packages are considered. Use make help
+# to see what these are.
+#
+
+include Makefile.help
 
 # Make configuration:
 SHELL := /bin/bash
 GNUMAKEFLAGS := --no-print-directory
 SUBMAKEFLAGS :=
 
-define assert_numeric_bool
-$(if $(filter-out 0 1,${$1}), $(error Unexpected value for parameter $1, expect either 0 or 1))
-endef
+META_PKG := cloe
+PLUGIN_PKGS := $(wildcard plugins/*)
+ALL_PKGS := fable runtime models oak engine ${PLUGIN_PKGS} ${META_PKG}
+WITHOUT_PKGS :=
```

Comment:
```suggestion
WITHOUT_PKGS := $(wildcard optional/*)
```

Reply:
These aren't included anyway in `PLUGIN_PKGS` so it would have no effect.
However, using `WTIH_PKGS=optional/vtd` would probably work.